diff --git a/Sources/CImGui/include/cimgui.h b/Sources/CImGui/include/cimgui.h
index 08e57db..02e373d 100644
--- a/Sources/CImGui/include/cimgui.h
+++ b/Sources/CImGui/include/cimgui.h
@@ -35,11 +35,14 @@
 #ifdef _MSC_VER
 typedef unsigned __int64 ImU64;
 #else
-//typedef unsigned long long ImU64;
+typedef unsigned long long ImU64;
 #endif
 
 
-#ifdef CIMGUI_DEFINE_ENUMS_AND_STRUCTS
+#ifndef CIMGUI_DEFINE_ENUMS_AND_STRUCTS
+ typedef struct ImGuiTableColumnSettings ImGuiTableColumnSettings;
+ typedef struct ImGuiTableCellData ImGuiTableCellData;
+ typedef struct ImGuiStackTool ImGuiStackTool;
 
 typedef struct ImDrawChannel ImDrawChannel;
 typedef struct ImDrawCmd ImDrawCmd;
@@ -2884,7 +2887,7 @@ struct SDL_Window;
 typedef union SDL_Event SDL_Event;
 #endif // CIMGUI_DEFINE_ENUMS_AND_STRUCTS
 
-#ifndef CIMGUI_DEFINE_ENUMS_AND_STRUCTS
+#ifdef CIMGUI_DEFINE_ENUMS_AND_STRUCTS
 typedef struct ImGuiStorage::ImGuiStoragePair ImGuiStoragePair;
 typedef struct ImGuiTextFilter::ImGuiTextRange ImGuiTextRange;
 typedef ImStb::STB_TexteditState STB_TexteditState;

diff --git a/Sources/ImGui/ImGui+Definitions.swift b/Sources/ImGui/ImGui+Definitions.swift
index 1514df1..ec5b8fb 100644
--- a/Sources/ImGui/ImGui+Definitions.swift
+++ b/Sources/ImGui/ImGui+Definitions.swift
@@ -5087,7 +5087,7 @@ public typealias ImSpanAllocator = OpaquePointer
 	return igImUpperPowerOfTwo(v)
 }
 
-@inlinable public func ImVectorreservediscard(_ this: UnsafeMutablePointer<ImVector>!, _ new_capacity: Int32) -> Void {
-	return ImVector_reserve_discard(this,new_capacity)
-}
+// @inlinable public func ImVectorreservediscard(_ this: UnsafeMutablePointer<ImVector>!, _ new_capacity: Int32) -> Void {
+// 	return ImVector_reserve_discard(this,new_capacity)
+// }
 
